packages/react/src/routing/OutletPageManager.tsx

```diff
@@ -24,6 +24,17 @@ export class OutletPageManager extends React.Component<OutletPageManagerProps> {
     super(props);
 
     this.outletIsReady = false;
+
+    /**
+     * This binds the scope of the following methods to the class scope.
+     * The `.bind` method returns a new function, so we need to assign it
+     * in the constructor to avoid creating a new function when removing the
+     * event listeners.
+     */
```

Comment:
```suggestion
```
I don't think this comment adds much for people revisiting the code in the future. Binding handlers in the constructor like this is a common React pattern, but calling attention to it makes it seem like we're doing something weird which is mildly confusing. You have to know that we used to bind them when adding the listeners in `componentDidMount` (which wasn't correct anyway) for this to make sense.

Reply:
Maybe we add more context on what pattern we're trying to advise against, in that case? Something like this:
```suggestion
    /**
     * This binds the scope of the following methods to the class scope.
     * The `.bind` method returns a new function, so we need to assign it
     * in the constructor rather than when adding or removing the listeners
     * to avoid creating a new function.
     */
```
At the moment, it's not clear unless you have the context of this PR's changes. I still think this is a common enough React pattern that we don't need a comment at all, and that adding one is more confusing than it is helpful, but my background is in React so I acknowledge I could be too biased 😄